.github/workflows/pypi-publish.yml

```diff
@@ -0,0 +1,29 @@
+name: Publish package to PyPi
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  push:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.6
+
+      - name: Install pip
+        run: pip install -r requirements/pip.txt
+
+      - name: Build package
+        run: python setup.py sdist bdist_wheel
+
+      - name: Publish to PyPi
+        uses: pypa/gh-action-pypi-publish@master
+        with:
+          user: edx
```

Comment:
we need to hold off on this, these settings don't seem to work

Reply:
Should we keep the PR on hold until it resolves? or skip this part for now?

Reply:
I think you just need to use `__token__` as the user